### ISSUE TITLE
Fatle crash of nvda in java applications

### ISSUE THREAD

**Issue by AlexandrKozlovskiy** [nvaccess/nvda#13190]
Hello nvda developers. I have nvda 2021.3 but it was before too. When i try select alot of code in android studio ide,nvda sometimes crash with some errors. I selected debug type of input and selected all imput categories in settings,but the last info,which i have in log,it what i press control+a. It seems,what in some versions of nvda in logs we have a reason of fatal crash,but now it not works. If i run nvda and focus in edit field with code,nvda crash again,and again no usefull info in logs. Nvda not crash,only if i before starting of nvda cancel selection of text on edit with code.

Another crash,which not fatal,but nvda not speak anymore,happen when we,for example,do click on some table cell in android studio,but it actual not for all cells. The crash in this case is:

DEBUG - NVDAObjects.JAB.JAB._get_states (03:02:06.046) - MainThread (10428):
states: enabled,focusable,visible,showing,armed,pressed
ERROR - eventHandler.executeEvent (03:02:06.047) - MainThread (10428):
error executing event: gainFocus on <NVDAObjects.JAB.TableCell object at 0x00D7F470> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "NVDAObjects\JAB\__init__.pyc", line 541, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1173, in event_gainFocus
  File "NVDAObjects\JAB\__init__.pyc", line 489, in reportFocus
  File "NVDAObjects\__init__.pyc", line 1028, in reportFocus
  File "speech\speech.pyc", line 555, in speakObject
  File "speech\speech.pyc", line 598, in getObjectSpeech
  File "speech\speech.pyc", line 450, in getObjectPropertiesSpeech
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\JAB\__init__.pyc", line 605, in _get_rowNumber
  File "JABHandler.pyc", line 564, in getAccessibleTableRow
ctypes.ArgumentError: argument 3: <class 'TypeError'>: wrong type

So,please fix this crash and do please,as it was before,to nvda save in log even reasons of fatal crash,as it was before.

**Comment by AlexandrKozlovskiy** [comment]
I forgot to add,what it's hapens on windows 11 64 bytes.

**Comment by Brian1Gaff** [comment]
Does the software support the access bridge? I try to steer clear of Java 
stuff as they seem to be laggy and often simple things like say all have 
unpredictable outcomes.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by mwhapples** [comment]
I use IntelliJ and related IDEs daily and have not encountered these issues. Probably needs more exact details on how to reproduce. However for that exception stack trace I do see how potentially that could be caused, but without being able to reproduce it I cannot figure out the best way to fix it.

**Comment by Adriani90** [comment]
@AlexandrKozlovskiy can you still reproduce this issue with NVDA 2024.1 or NVDA last alpha version?
You can download the last alpha version here:
https://www.nvaccess.org/files/nvda/snapshots/

**Comment by AlexandrKozlovskiy** [comment]
I am not sure,because i not updated android studio recently,and about selection of large text i am not sure too,but it seems,what with selection of big text i almost haven't this issue.

**Comment by Adriani90** [comment]
The initial author seem to not really have this issue anymore.
I am closing as works for me.
@AlexandrKozlovskiy if you are still impacted by this issue with the updated version of Android Studio and NVDA 2024.1 or NVDA last alpha, please open a new issue, and make sure you fill the complete bug template.
Also don't forget to attach a log file with log level set to debug in NVDA, and a crashdump if there is one crashdump generated for NVDA in %temp% or in %localappdata%.
